### PR TITLE
runtime: Update hypervisor generated code

### PR DESCRIPTION
**Pull request by stevenhorsman**
Update to use ttrpc_out instead of grpc_out

Fixes: #8519

### CODE REVIEW

**Review by beraldoleal — APPROVED** [review]
This lgtm, thanks. Let's see if CI will catch something.

**Review by GabyCT — APPROVED** [review]

**Comment by stevenhorsman** [comment]
We've got this working as best we can test with the cloud-api-adaptor with a re-vendor, so I think this is ready to go in now.

**Comment by stevenhorsman** [comment]
False alarm on the `do-not-merge` I thought something was missing, but I've checked and it's fine

File: src/runtime/virtcontainers/remote.go [hunk 1]
```diff
@@ -32,18 +32,20 @@ type remoteHypervisor struct {
 type remoteHypervisorSandboxID string
 
 type remoteService struct {
-	conn   *grpc.ClientConn
-	client pb.HypervisorClient
+	conn   net.Conn
+	client pb.HypervisorService
```
Comment (wainersm) [inline src/runtime/virtcontainers/remote.go:36]: @stevenhorsman HypervisorClient (struct) implements HypervisorService (Interface), so it compiles. Any special reason to replace the type from the struct to interface?

**Comment by wainersm** [comment]
`Static checks / build-checks (runtime, make vendor) (pull_request)` fail seems legit

**Comment by stevenhorsman** [comment]
> `Static checks / build-checks (runtime, make vendor) (pull_request)` fail seems legit

Sorry the `go mod tidy` I ran was in one of the debug commits I dropped, so I've re-run and pushed. Thanks for flagging it up

File: src/runtime/virtcontainers/remote.go (hunk 1 above)
Comment (wainersm) [reply src/runtime/virtcontainers/remote.go:36]: hmmm there is a hypervisorClient (lower case 'h') in https://github.com/kata-containers/kata-containers/pull/8520/commits/613c75ba8ce1deb78913ac0929776b2bdf98ac33#diff-0df604aba3c29a070a7ed40486ce8193ecb2a36c2dabf91bb29e7627cff969d8R52 . This should be a private date structure so never mind.

**Review by wainersm — APPROVED** [review]
Thanks @stevenhorsman !

**Pull request merged**